### ISSUE TITLE
Race condition: some symbols not returned

### ISSUE THREAD

**Issue by dt** [newhook/go-symbols#5]
```
defer func() {
  mutex.Lock()
  syms = append(syms, v.syms...)
  mutex.Unlock()
}()

defer wg.Done()
```
The wait group is unblocked by [the deferred Done()](https://github.com/newhook/go-symbols/blob/master/main.go#L222). Since that is deferred *after* the [deferred append of of the result](https://github.com/newhook/go-symbols/blob/master/main.go#L216), it is run first, meaning the group is unblocked before the result is ready.
